website/dbt-versions.js

```diff
@@ -0,0 +1,20 @@
+exports.versions = [
+  // {
+  //   version: "1.1",
+  //   EOLDate: "2023-01-01"
```

Comment:
@jtcohen6 I think we need this to be available for the beta, right?
```suggestion
  //   EOLDate: "2023-03-18"
```